DQM/SiStripMonitorHardware/src/SiStripSpyEventMatcher.cc

```diff
@@ -88,7 +82,7 @@ namespace sistrip {
   {
     size_t fileNameHash = 0U;
     //add spy events to the map until there are none left
-    source_->loopSequential(*eventPrincipal_,fileNameHash,std::numeric_limits<size_t>::max(),boost::bind(&SpyEventMatcher::addNextEventToMap,this,_1));
+    source_->loopOverEvents(*eventPrincipal_,fileNameHash,std::numeric_limits<size_t>::max(),boost::bind(&SpyEventMatcher::addNextEventToMap,this,_1));
```

Comment:
Yes.  It is controlled by two bool parameters, sequential and sameLumiBlock, both defaulting to false.